### PR TITLE
Adds autofill to the adjstationjob command.

### PR DESCRIPTION
**Pull request by QuietlyWhisper**
## About the PR

Adds autofill to the adjstationjob command.

**Media**

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
image

no cl

### CODE REVIEW

File: Content.Server/Administration/Commands/Station/AdjustStationJobCommand.cs [hunk 1]
```diff
@@ -57,4 +60,25 @@ public void Execute(IConsoleShell shell, string argStr, string[] args)
...
+
+        var stations = _entityManager
+            .System<StationSystem>()
+            .Stations
+            .Select(station =>
+            {
+                var meta = _entityManager.GetComponent<MetaDataComponent>(station);
+
+                return new CompletionOption(station.ToString(), meta.EntityName);
+            });
```
Comment (metalgearsloth) [inline Content.Server/Administration/Commands/Station/AdjustStationJobCommand.cs:80]: Can't you just use the component completionhelper and get stations?

File: Content.Server/Administration/Commands/Station/AdjustStationJobCommand.cs [hunk 2]
```diff
@@ -57,4 +60,25 @@ public void Execute(IConsoleShell shell, string argStr, string[] args)
...
+
+        return CompletionResult.FromHintOptions(stations, null);
```
Comment (metalgearsloth) [inline Content.Server/Administration/Commands/Station/AdjustStationJobCommand.cs:82]: Just use FromOptions if you don't want to hint.

File: Content.Server/Administration/Commands/Station/AdjustStationJobCommand.cs (hunk 1 above)
Comment (QuietlyWhisper) [reply Content.Server/Administration/Commands/Station/AdjustStationJobCommand.cs:80]: I really don't know, I just ported the code from crewmanifest

File: Content.Server/Administration/Commands/Station/AdjustStationJobCommand.cs (hunk 2 above)
Comment (QuietlyWhisper) [reply Content.Server/Administration/Commands/Station/AdjustStationJobCommand.cs:82]: I don't really know what that means, I just ported the code from crewmanifest

**Pull request closed without merging**